Pod/Classes/VOKCoreDataCollectionTypes.h

```diff
@@ -0,0 +1,61 @@
+//
+//  VOKCoreDataCollectionTypes.h
+//  Pods
+//
+//  Created by Carl Hill-Popper on 10/23/15.
+//
+//
+
+#ifndef VOKCoreDataCollectionTypes_h
+#define VOKCoreDataCollectionTypes_h
+
+#if __has_feature(objc_generics)
+//forward declare classes used here since we don't know where this will be included
+@class NSManagedObject;
+@class NSManagedObjectID;
+@class VOKManagedObjectMap;
+
+//include the pointer for this one since the fallback is id
+#define VOKManagedObjectSubclassPtr __kindof NSManagedObject *
```

Comment:
If I'm reading http://www.miqu.me/blog/2015/06/09/adopting-objectivec-generics/ correctly, `__kindof` allows pulling things out of the collection into subclass types without warnings, but putting subclass type objects into the collection shouldn't require `__kindof`?